routes/planReview.js

```diff
@@ -19,62 +25,57 @@ router.post("/api/planReview/request", auth, async (req, res) => {
   const reviewee_name = req.body.reviewee_id;
   const reviewer_id = req.body.reviewer_id;
   if (!plan_id || !reviewer_id || !reviewee_id) {
-    errorHandler(res, 400, {
-      message:
-        "Missing plan_id, reviewee_id or reviewer_id in the request body.",
-    });
-  } else if (req.user._id !== reviewee_id) {
-    forbiddenHandler(res);
-  } else {
-    const review = await planReviews
+    return missingHandler(res, { plan_id, reviewer_id, reviewee_id });
+  }
+  if (req.user._id !== reviewee_id) {
+    return forbiddenHandler(res);
+  }
+  try {
+    const _review = await Reviews
       .findOne({ reviewer_id, reviewee_id, plan_id })
       .exec();
-    if (review) {
-      errorHandler(res, 409, {
+    if (_review) {
+      return errorHandler(res, 409, {
         message: "Review request already created.",
       });
-    } else {
-      const planReview = {
-        reviewee_id,
-        reviewer_id,
-        plan_id,
-        requestTime: Date.now(),
-        status: "PENDING",
-      };
-      planReviews
-        .create(planReview)
-        .then(async (review) => {
-          postNotification(
-            `${reviewee_name} has requested you to review a plan.`,
-            [reviewer_id],
-            review._id,
-            "PLANREVIEW"
-          );
-          const reviewer = await users.findById(reviewer_id);
-          const reviewee = await users.findById(reviewee_id);
-          await sendReviewMail(
-            reviewee.name,
-            reviewer.name,
-            reviewer.email,
-            review._id,
-            res
-          );
-          returnData(review, res);
-        })
-        .catch((err) => errorHandler(res, 500, err));
     }
+    const planReview = {
+      reviewee_id,
+      reviewer_id,
+      plan_id,
+      requestTime: Date.now(),
+      status: "PENDING",
```

Comment:
Usually these constants should be put in an enum.
ie.
const PLAN_STATUS = {pending: "PENDING", review: "PLANREVIEW"}